pyseir/cli.py

```diff
@@ -108,5 +129,15 @@ def generate_state_report(state):
 
 @entry_point.command()
 @click.option('--state', default='', help='State to generate files for. If no state is given, all states are computed.')
-def run_all(state=None):
-    _run_all(state)
+@click.option('--output-interval-days', default='', type=int, help='Number of days between outputs for the WebUI payload.')
```

Comment:
What is output-interval-days? Time forward for forecasting?

Reply:
You can read the help string there. It specifies the output data interval that gets consumed by the Web UI.